src/components/Header.js

```diff
@@ -1,10 +1,77 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
-}
+const Header = ({ userList, toggleUser, status }) => {
+    const [userName, setUserName] = useState('고은');
+    const [userStatus] = useState('현재 활동중');
+    const [userImage, setUserImage] = useState(userList[1]);
+
+    const onClick = () => {
+        toggleUser();
+    };
+
+    // status 상태에 따라 프로필 사진이 바뀜
+    useEffect(() => {
+        if (status) {
+            setUserImage(userList[1]);
+            setUserName('고은');
+        } else {
+            setUserImage(userList[0]);
+            setUserName('정쿨');
+        }
+    }, [status]);
+
+    return (
+        <Wrapper>
+            <UserProfile onClick={onClick}>
```

Comment:
onClick함수내부에 toggleUser이란 함수만 있다면 요기에 바로 써도 될 것 같아요
```suggestion
            <UserProfile onClick={toggleUser}>
```

Reply:
그런 방법이 있었네요 ㅋㅋㅋㅋ 아직 미숙한가 봅니다 ㅠㅠ!!
감사합니다